packages/public-api/src/website/service.ts

```diff
@@ -106,15 +112,12 @@ export async function syncWebPageScanResult(scanReport: SystemApi.ScanReport) {
   });
 }
 
-function mapInternalWebsiteStatus(status: internalApi.WebPageScan.Status) {
+// TODO: add protected website use-case
```

Comment:
Do you intentionally want to merge this and fix after?

Reply:
Yes

Reply:
Task: https://github.com/fingerprintjs/gradejs-backend/issues/195